### ISSUE TITLE
Intermittent JSON parse failure

### ISSUE THREAD

**Issue by chris-araman** [mxcl/xcodebuild#10]
https://github.com/mxcl/xcodebuild/runs/2754225991
```
Run ./
  with:
    platform: macOS
    xcode: ^11
    working-directory: fixture
    code-coverage: false
    action: test
    quiet: true
Selected Xcode 11.7.0
Error: SyntaxError: Unexpected token b in JSON at position 93
```

https://github.com/chris-araman/CombineCloudKit/runs/2754115546
```
Run chris-araman/xcodebuild@tweaks
  with:
    action: build
    platform: tvOS
    xcode: ~11.3
    quiet: false
    code-coverage: false
  env:
    TOOLCHAINS: 
Selected Xcode 11.3.1
Error: SyntaxError: Unexpected token b in JSON at position 106
```

I wonder whether this might be related to this issue, [resolved in Xcode 12.5](https://developer.apple.com/documentation/xcode-release-notes/xcode-12_5-release-notes):
> xcodebuild no longer double-escapes the output of xcodebuild -showBuildSettings -json. (63554669)

**Comment by mxcl** [comment]
> I wonder whether this might be related to this issue, resolved in Xcode 12.5:
> 
> > xcodebuild no longer double-escapes the output of xcodebuild -showBuildSettings -json. (63554669)

Def. a contender.

I have enabled posting stdout for commands in debug mode (which is enabled for this repo) in #11, so hopefully we can catch the bad output.

**Comment by mxcl** [comment]
Obv. hasn't happened *since* 🙄

**Comment by mxcl** [comment]
https://github.com/mxcl/StreamReader/pull/4/checks?check_run_id=2765178199

**Comment by mxcl** [comment]
Caught it:

https://github.com/mxcl/StreamReader/pull/4/checks?check_run_id=2765489872

```json
{
  "project" : {
    "configurations" : [
      "Debug",
      "Release"
    ],
    "name" : "StreamReader",
    "schemes" : [
      "StreamReader-Package"
    ],
    "targets" : [
      "Path",
      "Path.swiftPackageDescription",
      "StreamReader",
      "StreamReaderPackageDescription"
    ]
  }
}
build session not created after 15 seconds - still waiting
```

**Comment by chris-araman** [comment]
@mxcl, it looks like I just hit this again. Maybe something like an additional newline? It's hard to tell from the log.
https://github.com/chris-araman/CombineCloudKit/runs/2782389615?check_suite_focus=true

**Comment by mxcl** [comment]
Probs fixed: #21

**Comment by mxcl** [comment]
I believe it will not happen anymore.
